packages/browser/src/transports/offline.ts

```diff
@@ -107,7 +107,7 @@ function createIndexedDbStore(options: BrowserOfflineTransportOptions): OfflineS
 
   // Lazily create the store only when it's needed
   function getStore(): Store {
-    if (store == undefined) {
+    if (store === undefined) {
```

Comment:
l: can we just make this:
```suggestion
    if (!store) {
```